templates/eap-cd-image-stream.json

```diff
@@ -46,6 +46,49 @@
                             "name": "registry.redhat.io/jboss-eap-7-tech-preview/eap-cd-openshift-rhel8:latest"
                         }
                     },
+                    {
+                        "name": "18",
+                        "annotations": {
+                            "description": "JBoss EAP continuous delivery Tech Preview version 18.",
+                            "iconClass": "icon-eap",
+                            "tags": "builder,eap,javaee,java,jboss,hidden",
+                            "supports": "eap:7.3,javaee:8,java:8",
```

Comment:
is `java:8` correct? I thought the image was supporting JDK 11?

I have the same question for EAP CD17 image actually as https://access.redhat.com/containers/?tab=tech-details#/registry.access.redhat.com/jboss-eap-7-tech-preview/eap-cd-openshift-rhel8 says:

> JAVA_VERSION=11.0

Reply:
@jmesnil yeah that should be 11, I'll update it and check the others too.